doc/setting-up.md

```diff
@@ -0,0 +1,159 @@
+Setting Up
+========
+part of [Qcode Documentation](index.md)
+
+* * *
+
+This document will guide through pulling together features from the qcode-tcl library to set up request handling for Naviserver that will include validation of user input and authenticating users. 
+
```

Comment:
This guide will demonstrate how to use our qcode-tcl library with Naviserver to handle incoming requests, perform validation of user input and perform user authentication